samples/CSharp/SourceGenerators/SourceGeneratorSamples/CSharpSourceGeneratorSamples.csproj

```diff
@@ -15,7 +15,7 @@
     <!-- Generator dependencies -->
     <PackageReference Include="CsvTextFieldParser" Version="1.2.2-preview" GeneratePathProperty="true" PrivateAssets="all" />
     <PackageReference Include="Handlebars.Net" Version="1.10.1" GeneratePathProperty="true" PrivateAssets="all" />
-    <PackageReference Include="Newtonsoft.Json" Version="13.0.1" GeneratePathProperty="true" PrivateAssets="all" />
+    <PackageReference Include="Newtonsoft.Json" Version="$(NewtonsoftJsonVersion)" GeneratePathProperty="true" PrivateAssets="all" />
```

Comment:
Should we update `NewtonsoftJsonVersion` to 13.0.3?

Reply:
There's basically a new 'revision' of Newtonsoft.Json each year, the last 3 years.  Sold.